src/Sylius/Bundle/ProductBundle/Doctrine/ORM/ProductVariantRepository.php

```diff
@@ -152,19 +171,18 @@ public function findByPhraseAndProductCode(string $phrase, string $locale, strin
 
     public function findByPhrase(string $phrase, string $locale): array
```

Comment:
What if we just add optional arguments here rather than creating a separate repository method? Especially as now the `findByPhrase` is not used at all